docs/src/components/ReactExample/index.tsx

```diff
@@ -85,8 +87,13 @@ const ReactExample = ({ exampleId, minHeight }: Props) => {
     })
     .join("\n");
 
+  const code = format(fields.example, {
+    parser: "typescript",
+    plugins: [parserTypeScript],
+  });
```

Comment:
This achieves the desired effect, but is it possible to do it in build time, rather than client-side? That way we can avoid adding prettier to the bundle.

Reply:
done